multinode-demo/remote_leader.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+
+mkdir -p ~/.ssh ~/solana ~/.cargo/bin
```

Comment:
replicode with remote_validator.sh

how about something that fetches our dependencies, instead?

Reply:
Validator is fetching most dependencies from leader node ($1 is leader IP here).
One option to remove replicode is to split the overlap portion to a separate (remote_node.sh?) script. Are you thinking something beyond that?